File: C/1035_Selection_Test_1.c
#include <stdio.h>

const char *result(int a, int b, int c, int d)
{
    if (b > c && d > a && (c + d) > (a + b) && c > 0 && d > 0 && a % 2 == 0)
        return "Valores aceitos";
    return "Valores nao aceitos";
}

int main()
{
    int a, b, c, d;
    scanf("%d %d %d %d", &a, &b, &c, &d);
    printf("%s\n",result(a,b,c,d));
    return 0;
}

File: Python/1035_Selection_Test_1.py
input = input().split(' ')

a = int(input[0])
b = int(input[1])
c = int(input[2])
d = int(input[3])

if(b > c and
    d > a and 
    (c + d) > (a + b) and
    c > 0 and 
    d > 0 and
    a % 2 == 0):
    print("Valores aceitos")
else :
    print("Valores nao aceitos")